json_format.py

```diff
@@ -0,0 +1,11 @@
+def handler_json():
```

Comment:
all functionality is expected to be put at 'metafoam' folder (future python package).
For now, this file can be removed from the pull request (no code coverage)